lib/writer/eyeD3.js

```diff
@@ -186,6 +186,10 @@ Writer.prototype.write = function(file, version, meta, callback)
                 break;
             case 'year':
                 args.push('-Y ' + value);
+                args.push('--orig-release-date', + value);
```

Comment:
Unfortunately eyeD3 0.6 can't handle the new arguments, thats why the tests failing.
Please wrap the new `--*-date` args block into a `if ('0.7' === this.version)` condition.